README.md

```diff
@@ -1,17 +1,17 @@
-# loadero-nightwatch-commands
+# loadero-nightwatch-commands ![npm](https://img.shields.io/npm/v/loadero-nightwatch-commands)
```

Comment:
Should add URL to https://www.npmjs.com/package/loadero-nightwatch-commands I think. Currently clicking on the badge just opens the image in a new tab.